hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md

```diff
@@ -80,7 +80,7 @@ message Board {
 Note that you will also have to modify the `x/leaderboard/genesis.go`. In it, look for:
 
 ```golang
-// Set if defined
+    // Set if defined
     if genState.Board != nil {
         k.SetBoard(ctx, *genState.Board)
     }
```

Comment:
```suggestion
// Set if defined
if genState.Board != nil {
    k.SetBoard(ctx, *genState.Board)
}
```
Why not like this?

Reply:
Just to indicate that this piece of code belongs to some function.